habitat/sims/habitat_simulator.py

```diff
@@ -451,3 +454,18 @@ def distance_to_closest_obstacle(self, position, max_search_radius=2.0):
         return self._sim.pathfinder.distance_to_closest_obstacle(
             position, max_search_radius
         )
+
+    @property
+    def up_vector(self):
+        return np.array([0.0, 1.0, 0.0])
+
+    @property
+    def forward_vector(self):
+        return -np.array([0.0, 0.0, 1.0])
+
+    def get_shortest_path_points(self, position_a, position_b):
```

Comment:
Let's create method similar to this: https://github.com/facebookresearch/habitat-api/blob/master/habitat/core/simulator.py#L288 and call it `shortest_path` that returns
ShortestPathPoints.

Reply:
We should name it `straight_shortest_path`.  If the shortest path is calculated on an occupancy grid via A*, the greedy follower will definitely not work as expected and will generate very strange paths.  You'd get much better paths from the code version that to estimates the geodesic gradient and follows that.

Reply:
I changed the name and updated the docs. Maybe instead of straight it should be geodesic though?

Reply:
@erikwijmans, can you elaborate more what `straight` stands for here and how it's different from path used for geodesic distance calculation?